doc/build-osx.md

```diff
@@ -16,7 +16,8 @@ Then install [Homebrew](http://brew.sh).
 Dependencies
 ----------------------
 
-    brew install automake berkeley-db4 libtool boost --c++11 miniupnpc openssl pkg-config protobuf --c++11 qt5 libevent
+    brew install automake libtool boost --c++11 miniupnpc openssl pkg-config protobuf --c++11 qt5 libevent
+    brew install berkeley-db # You need to make sure you install a version >= 5.1.29, but as close to 5.1.29 as possible. Check the homebrew docs to find out how to install older versions.
```

Comment:
Do we want to pull docs changes so early? I think it would be beneficial to keep most of the cosmetic details separate as we did before. Makes picking changes later easier I guess? But I'm open for anything ;)